azurelinuxagent/common/event.py

```diff
@@ -581,24 +625,22 @@ def _trim_extension_event_parameters(event):
         event.parameters = trimmed_params
 
     @staticmethod
-    def report_dropped_events_error(count, errors, op, max_errors_to_report): # pylint: disable=C0103
+    def report_dropped_events_error(count, errors, operation_name, max_errors_to_report):
         err_msg_format = "DroppedEventsCount: {0}\nReasons (first {1} errors): {2}"
         if count > 0:
-            add_event(op=op,
+            add_event(op=operation_name,
                       message=err_msg_format.format(count, max_errors_to_report, ', '.join(errors)),
                       is_success=False)
 
-    def collect_events(self): # pylint: disable=R0914
+    # too-many-locals<R0914> Disabled: The number of local variables is OK
```

Comment:
we should just disable this warning globally :)

Reply:
Haha I agree! Let me do it in another PR though just to keep both the issues isolated.